ui/job-view/pushes/Push.jsx

```diff
@@ -177,7 +203,11 @@ class Push extends React.PureComponent {
       const newIds = jobs.map(job => job.id);
       // remove old versions of jobs we just fetched.
       const existingJobs = jobList.filter(job => !newIds.includes(job.id));
-      const newJobList = [...existingJobs, ...jobs];
+      // Join both lists and add test_paths property
+      const newJobList = [...existingJobs, ...jobs].map(job => {
+        job.test_paths = jobTypeNameToManifests[job.job_type_name] || [];
```

Comment:
I believe this could be shortened to:

    const newJobList = [...existingJobs, ...jobs].map(job => ({
        ...job, test_paths: jobTypeNameToManifests[job.job_type_name] || [],
    });

And no ``return`` needed.